metadata/metadata_service/proxy/neo4j_proxy.py

```diff
@@ -1815,3 +1816,23 @@ def get_feature(self, *, feature_uri: str) -> Feature:
             created_timestamp=feature_metadata['created_timestamp'],
             watermarks=feature_metadata['watermarks'])
         return feature
+
+    def get_resource_generation_code(self, *, uri: str, resource_type: ResourceType) -> Query:
+        """
+        Executes cypher query to get query nodes associated with resource
+        """
+        neo4j_query = textwrap.dedent("""\
+        MATCH (feat:{resource_type} {{key: $resource_key}})
```

Comment:
Is there an extra {} around {key: $resource_key}?

Reply:
this is necessary when you have string interpolation and when making a cypher query where you create a node you set the properties by using {}. We do this for other queries like: https://github.com/amundsen-io/amundsen/blob/main/metadata/metadata_service/proxy/neo4j_proxy.py#L355